### PR TITLE
Don't include unknown files in the updates image

### PR DESCRIPTION
**Pull request by poncovka**
For example, Docker files and Git Hub workflows.

### CODE REVIEW

**Review by martinpitt — DISMISSED** [review]
Makes sense, thank you!

File: scripts/makeupdates [hunk 1]
```diff
@@ -289,19 +289,10 @@ def copy_updated_files(tag, updates, cwd, builddir):
...
-            fields = gitfile.split('/')
-            subdir = fields[0]
-            if subdir in ['po', 'scripts','command-stubs', 'tests',
-                          'docs', 'fonts', 'utils',
-                          'liveinst', 'dracut', 'data']:
-                continue
-            else:
-                sys.stdout.write("Including %s\n" % (gitfile,))
-                install_to_dir(gitfile, tmpupdates)
         else:
```
Comment (jkonecny12) [inline scripts/makeupdates:292]: What about removing this whole `else` part?

**Review by jkonecny12 — APPROVED** [review]
Looks good to me otherwise.
Thanks for this!

Don't we want to remove also ``configure.ac`` from the ignores above?

File: scripts/makeupdates (hunk 1 above)
Comment (poncovka) [reply scripts/makeupdates]: Fixed.

**Comment by poncovka** [comment]
Updated.

**Review by jkonecny12 — APPROVED** [review]
LGTM!

**Pull request merged**